card/mod/01_core/set/all/collection.rb

```diff
@@ -48,8 +48,13 @@ def item_type
   nil
 end
 
-def include_item? cardname
-  item_names.map{|name| name.to_name.key}.member? cardname.to_name.key
+def include_item? item
+  key = if Card === item
+    item.cardname.key
+  else
+    item.to_name.key
+  end
+  item_names.map{|name| name.to_name.key}.member? key
```

Comment:
not an issue for this release (you didn't change this), but it would be nice to use find here.  (stop looking when you find it)